grpc/rust/compile/compile.bzl

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
Algorithm:

1. Collect the protos (`.proto` sources) and the `protoc` executable, declaring them as `inputs`;
2. Declare one `output` file for each unique Protobuf `package` in the proto sources (for Protocol, there is just one package, `typedb.protocol`)
3. Run a Rust binary that uses the `tonic_build` crate to build the Rust gRPC + Protobuf bindings.